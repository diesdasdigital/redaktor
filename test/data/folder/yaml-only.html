{
  "title": "only yaml",
  "markdown": "",
  "path": "folder/yaml-only",
  "updatedAt": "2020-08-28T12:38:38.009Z",
  "createdAt": "2020-08-28T12:38:38.009Z"
}