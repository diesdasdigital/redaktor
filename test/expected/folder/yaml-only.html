<pre>{
  "title": "only yaml",
  "markdown": "",
  "path": "folder/yaml-only"
}</pre>